shub/image/push.py

```diff
@@ -177,3 +155,33 @@ def show(self):
         self.total_bar.close()
         for bar in self.layers_bars.values():
             bar.close()
+
+
+def _create_total_bar():
+    return utils.ProgressBar(
+        total=1,
+        desc='total',
+        file=click.get_text_stream('stdout'),
+        dynamic_ncols=True,
+        miniters=1,
+        # don't need rate here, let's simplify the bar
+        bar_format='{l_bar}{bar}| {n_fmt}/{total_fmt}'
+    )
```

Comment:
I think these should be methods in the class - in case someone would like to change the behavior and override them